setup.py

```diff
@@ -62,7 +62,7 @@ def run(self):
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
     packages=['babel', 'babel.messages', 'babel.localtime'],
-    package_data={'babel': ['global.dat', 'localedata/*.dat']},
+    package_data={'babel': ['global.dat', 'locale-data/*.dat']},
```

Comment:
Since you already included those in `MANIFEST.in`, and we are using *setuptools*, you can save some work by using `include_package_data=True` instead.  [Reference](http://pythonhosted.org/setuptools/setuptools.html#new-and-changed-setup-keywords).